tests/vm_backup_test.go

```diff
@@ -30,8 +30,6 @@ var _ = Describe("[smoke] VM Backup", func() {
 
 	var r = framework.NewKubernetesReporter()
 
-	const snapshotLocation = "test-location"
```

Comment:
I see that `CreateBackupForNamespace` (called multiple times in this file) is still using `snapshotLocation`. I guess it works because it uses the one defined in `dv_backup_test.go`, but isn't that confusing? Maybe it should be defined in utils or in both dv_backup_test.go and vm_backup_test.go to improve readability.

Reply:
right, that can be improved

Reply:
right, moved to utils,